### PR TITLE
Move the contribution testing guide to the confluence wiki

### PR DESCRIPTION
**Pull request by ajamato**
**Please** add a meaningful description for your change here

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by ajamato** [comment]
I completed #1 and #2, I am not too sure how to complete #3, happy to do so if there are some instructions, I am not too sure how to do it on this particular web framework

**Comment by kennknowles** [comment]
R: @melap might know. I think we have done this before. Last time we removed the "Team" page and linked directly to the ASF directory, I think I just deleted it, which caused search engine results to be a 404 for a while. In this case, doubtful anyone really cares.

**Comment by melap** [comment]
I think you can do it, but it would mean leaving the md file in the repo with the redirect, so the tradeoff would repo cleanliness in the form of an empty file in the repo (described below). 

if you do want to pursue redirection, I vaguely recall the redirect_from plugin does support redirection to external links, so it would mean leaving the existing file in the repo, making it empty except the header, and add a redirect_to like this (might have to use some sort of quoting with the url as it has + but I am not sure without testing it);

redirect_to:
  - https://cwiki.apache.org/confluence/display/BEAM/Contribution+Testing+Guide

**Comment by ajamato** [comment]
@melap This seems to work now, thanks for teaching me how. :)

File: website/src/contribute/testing.md [hunk 1]
```diff
@@ -3,428 +3,18 @@ layout: section
 title: 'Beam Testing'
 section_menu: section-menu/contribute.html
 permalink: /contribute/testing/
+redirect_from:
+  - /contribute/testing/
```
Comment (kennknowles) [inline website/src/contribute/testing.md]: I don't think you need this. The use of `redirect_from` is to specify a separate URL from which to bring the user to this page.

Comment (ajamato) [reply website/src/contribute/testing.md]: Done, rm'd and confirmed that it works without redirect_from

**Comment by ajamato** [comment]
Run Website PreCommit

**Comment by ajamato** [comment]
retest this please

**Comment by ajamato** [comment]
Run Website PreCommit

**Comment by ajamato** [comment]
beam_14 seems to be in a bad state, and failing due to docker not being setup properly. Retesting until it goes on another machine

**Comment by ajamato** [comment]
Run Website PreCommit

**Comment by ajamato** [comment]
Run Website PreCommit on beam13

**Comment by ajamato** [comment]
Run Website PreCommit

**Comment by ajamato** [comment]
Run Website PreCommit

**Comment by ajamato** [comment]
I misread the error, and its due to the precommit page going down, failing the link validator.
    :url_ignore => [
        /104.154.241.245/  # Preccommit dashboard is down [BEAM-6455].
        # To ignore link checking for a URL, i.e. if it is temporariliy down,
        # add a URL regex below with a tracking JIRA issue. For example:
        # /example.com/, # BEAM-1234 failing due to expired SSL cert
    ],

**Comment by ajamato** [comment]
All resolved now :). @kennknowles Would you please squash and commit?

**Comment by kennknowles** [comment]
Thanks for working through all this. I filed https://issues.apache.org/jira/browse/BEAM-6458 for the raw i.p. thing.

**Pull request merged**